.travis.yml

```diff
@@ -6,6 +6,10 @@ python:
     - "3.4"
     - "3.5"
 
+install:
+  - pip install -r requirements.txt
```

Comment:
In this project I consider `requirements.txt` as dev requirements. The actual requirements necessary to install the software depend on the installation method. They are either in `setup.py` or in the deb package meta-data.

So you can just add your dependency to `requirements.txt`.